addons/l10n_es/data/account.account.tag.csv

```diff
@@ -0,0 +1,16 @@
+"id","name","applicability","country_id/id"
+"account_tag_mod390_p103","+mod390[103]","taxes","base.es"
```

Comment:
Why can not that be automatically be there from the report line?  What happens if this file is not there at all?  (I think if you update CoA without updating the module, you have a problem anyways)